### PR TITLE
add section for ct tests in testing pyramid lesson

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: content/courses/testing-foundations/the-testing-pyramid.mdx [hunk 1]
```diff
@@ -20,13 +20,21 @@ So again, you are simply testing that given a set of inputs into this function,
...
 
-The point of integration testing is to ensure that individual pieces or units within an application work together as expected.
+Integration testing aims to ensure that individual pieces or units within an application work together as expected.
 
 Unlike unit tests, which should always be independent, integration tests are fundamentally dependent. Their entire purpose is to test the dependencies of pieces within a system are working together correctly.
 
 So the distinction between a unit test and an integration test is that unit tests test things in isolation, and integration tests, are testing portions of your application that are related to one another, never in isolation.
 
-Integration tests are great for when you are building a new application from scratch or an application or feature that does not have a UI yet. However, once you have a UI, that is when you should be writing end to end (E2E) tests.
+Integration tests are great for building a new application from scratch or an application or feature that still needs a UI. However, once you have a UI, consider component and end-to-end tests.
```
Comment (jaffrepaul) [inline content/courses/testing-foundations/the-testing-pyramid.mdx:29]: What are we getting at when we say these tests are good for something that _still needs_ a UI? I found that kind of weird.

**Review by jaffrepaul — COMMENTED** [review]
This is a nice addition 👍

I added one question about a sentence that was already in place that might use a tweak.

File: content/courses/testing-foundations/the-testing-pyramid.mdx (hunk 1 above)
Comment (elylucas) [reply content/courses/testing-foundations/the-testing-pyramid.mdx:29]: Here it seems the original author thought that integration tests didn't test UI, so its time to look at CT and e2e for that.

Comment (jaffrepaul) [reply content/courses/testing-foundations/the-testing-pyramid.mdx:29]: Yeah, seems odd to posit that part about the UI. Remember that pre v10 scaffolded out an _/integrations_ folder for tests. I think it's effective if we just took out that _or an application or feature that still needs a UI_

What do you think?

Comment (elylucas) [reply content/courses/testing-foundations/the-testing-pyramid.mdx]: Ya, I went ahead and just removed the sentence.

**Pull request merged**